veniq/dataset_collection/dataflow/data_aggregation.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+
+import pandas as pd
+from pandas import DataFrame
+
+from veniq.dataset_collection.dataflow.annotation import InvocationType
+from veniq.dataset_collection.types_identifier import InlineTypesAlgorithms
+from bonobo.config import Exclusive
+
+
+def aggregate(dirs, dct):
```

Comment:
can u come up with a more informative name of the method, and also type-annotate it?

Reply:
what does this methid do?